yaml_files/simphony_metadata.yml

```diff
@@ -454,22 +454,24 @@ CUDS_KEYS:
 
   DIRICHLET:
     definition: Dirichlet boundary condition to specify the value the solutions takes on the boundary of the domain.
-    parent: DIFFERENTIAL_EQUATION_CONDITION
+    parent: FIELD_CONDITION
 
   NEUMANN:
     definition: Neumann boundary condition, it specifies the values that the derivative of a solution with respect to the variable is to take on the boundary of the domain.
-    parent: DIFFERENTIAL_EQUATION_CONDITION
+    parent: FIELD_CONDITION
 
   ROBIN:
-    parent: DIFFERENTIAL_EQUATION_CONDITION
+    parent: FIELD_CONDITION
     definition: A mixed boundary condition $\alpha \Phi (x) + \beta (x) \partial {\Phi} / \partial {\bf{n}} (x) = h(x)$, with $h$ is the value (CUBA.VALUE).
     CUBA.DIRECHLET:
     CUBA.NEUMANN:
 
   EMPTY:
-    definition: Empty boundary condition
-    parent: CUBA.VALUE_CONDITION
-    value: CUBA.NONE
+    definition: an entity to represent that no boundary condition is applied on that domain
+    parent: CUBA.CONDITION
+    CUBA.VALUE:
+      default: CUBA.NONE
+      scope: CUBA.SYSTEM
```

Comment:
you can't have a CUBA.SYSTEM _and_ a default. It makes no sense. Either you have a default, or you let the compiler decide for it. They are mutually exclusive.

```
262     - ``default``:
263         Indicates the default value for the property once the ``CUDS Item`` has 
264         been instantiated.
265         The default MUST be type compatible with the property entry key
266         (eg. integers if the data is an integer)
267         If the key refers to a CUBA data, the default must match shape, type and length
268         requirements specified for the CUBA data, keeping into account the shape of the CUBA data
269         itself. 
270         If the key refers to a CUDS item, the default must belong to the hierarchy defined by
271         the CUDS item designated in the key.
272         if the ``scope`` is ``CUBA.SYSTEM``, this key MUST NOT be present

```